drizzle/migrations/0005_fix_chat_db_fk_normalization.sql

```
@@ -0,0 +1,73 @@
+  DROP CONSTRAINT IF EXISTS visualizations_user_id_fkey;
+
+ALTER TABLE public.chat_participants
+  DROP CONSTRAINT IF EXISTS chat_participants_user_id_fkey;
+
+ALTER TABLE public.prompt_generation_jobs
+  DROP CONSTRAINT IF EXISTS prompt_generation_jobs_user_id_fkey;
+
+-- Step 2: Add new foreign key constraints referencing public.users.id
+ALTER TABLE public.chats
+  ADD CONSTRAINT chats_user_id_fkey
+    FOREIGN KEY (user_id) REFERENCES public.users(id) ON DELETE CASCADE;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)